src/Transport/EventRouting/MigrationTopology.cs

```diff
@@ -188,7 +188,7 @@ protected override string GetPublishDestinationCore(string eventTypeFullName)
     internal override SubscriptionManager CreateSubscriptionManager(SubscriptionManagerCreationOptions creationOptions) => new MigrationTopologySubscriptionManager(creationOptions, Options);
 
     internal const string ObsoleteMessage =
-        "The migration topology is intended to be used during a transitional period, facilitating the migration from the single-topic topology to the topic-per-event topology. The migration topology will eventually be phased out over subsequent releases. Should you phase challenges during migration, please reach out to https://github.com/Particular/NServiceBus.Transport.AzureServiceBus/issues/1170.";
+        "The migration topology is intended to be used during a transitional period, facilitating the migration from the single-topic topology to the topic-per-event topology. The migration topology will eventually be phased out over subsequent releases. Should you phase challenges during migration, please reach out to |https://github.com/Particular/NServiceBus.Transport.AzureServiceBus/issues/1170|.";
```

Comment:
That `.` at the end, is that needed?

TIL: Using | to get links in obsolete messages

Reply:
The . makes it a real sentence, so I'd prefer to keep it.

Reply:
I get it. the || are the equivalent of markdown its <> syntax